public/js/redux/chartsRedux.js

```diff
@@ -36,7 +36,7 @@ const chartsRedux = State({
 
     updateComposerVsIncopy(state, { chartData, startDate, endDate }) {
         const { composerResponse, inCopyResponse } = chartData;
-        const range = endDate.diff(startDate, 'days');
+        const range = endDate.diff(startDate, 'days') + 1;
```

Comment:
why this change?

Reply:
this is to fix the condition by which the `fillMissingDates` is called, but it's probably more correct to do change ```const composerData = composerResponse.data.length < range ?  fillMissingDates(...)``` to ```const composerData = composerResponse.data.length <= range ?  fillMissingDates(...)```